docs/source/openvino/optimization.mdx

```diff
@@ -166,6 +166,26 @@ calibration_dataset = quantizer.get_calibration_dataset(
 The `quantize()` method applies post-training static quantization and export the resulting quantized model to the OpenVINO Intermediate Representation (IR). The resulting graph is represented with two files: an XML file describing the network topology and a binary file describing the weights. The resulting model can be run on any target Intel device.
 
 
+#### Whisper Quantization
```

Comment:
Should not it be `Speech-to-text Models Quantization`

Reply:
Renamed. The reason I named it Whisper is because as I understand it is the only speech-to-text model currently supported.
cc @eaidova